pkg/tokens/builtin/server/model/types.go

```diff
@@ -0,0 +1,16 @@
+package model
```

Comment:
I like the idea of a dedicated package for API types, but it name is inconsistent with other code. 

Names already in place:

* `pkg/xds/bootstrap/rest`
* `pkg/api-server/types`

And now:

* `pkg/tokens/builtin/server/model`

Let's pick a single convention

Reply:
picked `/types` and changed all those 3 to this convention